### PR TITLE
Added A1Z26 Cipher

### PR DESCRIPTION
**Pull request by LethargicLeprechaun**
### **Describe your change:**

* [x] Add an algorithm?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: ciphers/a1z26.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+def encode(plain : str) -> list:
+    """
+    >>> encode("myname")
+    [13, 25, 14, 1, 13, 5]
+    """
+    result = []
+    for elem in plain:
+        result.append(ord(elem) - 96)
+    return result
```
Comment (cclauss) [inline ciphers/a1z26.py:18]: Please try to replace lines 15 to 18 with a single line using a `list comprehension`.

Comment (LethargicLeprechaun) [reply ciphers/a1z26.py]: Done.

File: ciphers/a1z26.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+def decode(encoded : list) -> str:
+    """
+    >>> decode([13, 25, 14, 1, 13, 5])
+    'myname'
+    """
+    result = ""
+    for elem in encoded:
+        result += chr(elem + 96)
+    return result
```
Comment (cclauss) [inline ciphers/a1z26.py:25]: Please try to replace lines 25 thru 28 with a one liner using str.join() of a Python generator.

Comment (LethargicLeprechaun) [reply ciphers/a1z26.py]: Also done.

**Comment by cclauss** [comment]
Nice work!  Thanks for your contribution!!

**Pull request merged**